### PR TITLE
Remove unneeded data that is now conveyed via generic Resource attrib…

### PR DESCRIPTION
**Pull request by tigrannajaryan**
…utes

The PR https://github.com/open-telemetry/opentelemetry-specification/pull/363/files
defines semantic conventions for library, host and service. There is no longer
need to hard code this information as protocol fields.

### CODE REVIEW

File: opentelemetry/proto/agent/common/v1/common.proto [hunk 1]
```diff
@@ -49,49 +43,9 @@ message Node {
 // Identifier that uniquely identifies a process within a VM/container.
 message ProcessIdentifier {
```
Comment (bogdandrutu) [inline opentelemetry/proto/agent/common/v1/common.proto]: I don't think this is needed as well.

File: opentelemetry/proto/agent/common/v1/common.proto [hunk 2]
```diff
@@ -33,12 +33,6 @@ message Node {
   // Identifier that uniquely identifies a process within a VM/container.
```
Comment (bogdandrutu) [inline opentelemetry/proto/agent/common/v1/common.proto]: Can we remove all the Node?

Comment (tigrannajaryan) [reply opentelemetry/proto/agent/common/v1/common.proto]: Yes. I will do in separate PR.

File: opentelemetry/proto/agent/common/v1/common.proto (hunk 1 above)
Comment (tigrannajaryan) [reply opentelemetry/proto/agent/common/v1/common.proto]: Correct, was planning to do in separate PR.

**Comment by bogdandrutu** [comment]
Please rebase

**Pull request merged**